tendrl/ceph_integration/manager/user_request.py

```diff
@@ -237,6 +237,43 @@ def _submit(self, commands=None):
         return ceph.rados_commands(self.fsid, self._cluster_name, commands)
 
 
+class ECProfileRequest(UserRequest):
```

Comment:
For ceph_command I agree. May be will rename the class as CephRequest but somehow the name looks too generic to me.
@r0h4n comments?